packages/radix-ui-themes/src/components/inset.tsx

```diff
@@ -32,19 +32,23 @@ const Inset = React.forwardRef<InsetElement, InsetProps>((props, forwardedRef) =
         className,
         withBreakpoints(side, 'rt-r-side'),
         withBreakpoints(clip, 'rt-r-clip'),
-        withBreakpoints(p, 'rt-r-p'),
-        withBreakpoints(px, 'rt-r-px'),
-        withBreakpoints(py, 'rt-r-py'),
-        withBreakpoints(pt, 'rt-r-pt'),
-        withBreakpoints(pr, 'rt-r-pr'),
-        withBreakpoints(pb, 'rt-r-pb'),
-        withBreakpoints(pl, 'rt-r-pl'),
+        withBreakpoints(p, 'rt-r-p', paddingMap),
+        withBreakpoints(px, 'rt-r-px', paddingMap),
+        withBreakpoints(py, 'rt-r-py', paddingMap),
+        withBreakpoints(pt, 'rt-r-pt', paddingMap),
+        withBreakpoints(pr, 'rt-r-pr', paddingMap),
+        withBreakpoints(pb, 'rt-r-pb', paddingMap),
+        withBreakpoints(pl, 'rt-r-pl', paddingMap),
         withMarginProps(marginProps)
       )}
     />
   );
 });
 Inset.displayName = 'Inset';
 
+const paddingMap = {
+  current: 'inset',
+};
```

Comment:
I changed `rt-r-p-current` classname to `rt-r-p-inset` so the name of the class reflects the implementation (e.g. in case we have another component unrelated to Inset with "current" padding value)